README.md

```diff
@@ -85,9 +85,14 @@ Once you complete the setup, you'll be able to run the various commands provided
 
 ## Build locally
 
-1. From the root `/tabler` directory, run `npm run start` in the command line.
-2. Open [http://localhost:3000](http://localhost:3000) in your browser, and voilà.
-3. Any change in the `/src` directory will build the application and refresh the page.
+You need to have `npm` and `bundler` installed.
+
+1. From the root `/tabler` directory, run installation in the command line:
+  - `npm install` 
+  - `bundler install` 
+2. Then execute `npm run start-plugins` to start up the application stack.
```

Comment:
without the `-plugins` the vendor files were not build and I didn't check further if that was some issue on my end